handbook/engineering/developer-insights/code-insights/processes.md

```diff
@@ -16,28 +16,79 @@ Attendees are encouraged to add comments on talking points asynchronously before
+
+  - Teammates can **_reorder_ proposed issues** on the ["Next iteration" board](https://github.com/orgs/sourcegraph/projects/200/views/4) before the mid-iteration sync to their liking. The order at the time of the mid-iteration Monday sync is the _proposed order_.
+
+  - At the mid-iteration [Monday sync](#weekly-sync), we take a look at the proposed issues together on the ["Next iteration" view](https://github.com/orgs/sourcegraph/projects/200/views/4) to **reach agreement** on the **set** of iteration issues, their **assignees**, and **order**, taking into account our [goals and roadmap](./goals.md), technical aspects, estimates, workloads on individuals, and release dates (the 20th of every month).<br>
+    As a rule of thumb, the sum of estimates for each individual should not exceed **8 days** (out of a 10-work-day iteration) so we have enough buffer, time to reply to asks from other teams, to plan the next iteration's projects, and self-organized slack time.
+
+- During an iteration, teammates **work on their assigned issues for the iteration in the order they are listed** in the ["Current iteration" view](https://github.com/orgs/sourcegraph/projects/200/views/1) of the board. When starting work on a task, the teammate **updates its status column to "In Progress"** to communicate it to their team. This gives a good overview in the ["Current iteraton" view](https://github.com/orgs/sourcegraph/projects/200/views/1), which can also be viewed in [Kanban layout](https://github.com/orgs/sourcegraph/projects/200/views/1?layout=board), on how the iteration is tracking.
+
+- If one or more issues that were planned for an iteration are looking to **not get finished** in the [current iteration](https://github.com/orgs/sourcegraph/projects/200/views/1) (while maintaining sustainable work practices) the assignee **raises this as soon as possible asynchronously** to the team (including the PM and EM), e.g. on the GitHub issue or Slack. These issues then **become _proposed_ issues for the next iteration** (meaning nothing carries over automatically, but we also don't just drop and forget missed issues).
+
+### Releases
+
+Despite following two-week iterations, our [releases are monthly on the 20th](../../releases.md#releases) and we may sometimes need to order tasks in a way that we can get important projects into the next release.
+
+We also intentionally plan so we can avoid merging significant work less than two days before a release (if a release is on the 20th, our last day to merge is the 18th). Exceptions require explicit approval of both the PM and EM.
+
+### Planning larger projects
+
+Larger (usually multi-iteration) projects coming from our roadmap go through three phases:
+
+1. **Product context and scope agreement** (usually 1-2 weeks)<br>
+   The PM creates a product-level RFC summarizing user desires from feedback and proposes a reasonable timeline to reach a scope agreement (mentioned in the RFC).
```

Comment:
```suggestion
   The driving individual creates a product-level RFC summarizing user desires from feedback and proposes a reasonable timeline to reach a scope agreement (mentioned in the RFC).
```
In some cases it could be Felix or Alicja, and in engineering-focused major projects I imagine anyone on the team might lead these efforts? Or do you want to specify "larger projects" = "larger product features"?

Reply:
Hmm I was thinking back and forth on this. My primary motivation was to basically document what you were already doing for larger "product features": "What are these RFCs that Joel always does? What are they for? What am I expected to do with it? What comes after it?"

At the same time, there can absolutely be "larger" features that go through similar phases that are driven by engineering or design, my intend is by no means to prohibit or discourage that.
However, in those cases the expectations here would also not be fitting – I wouldn't expect an engineering RFC to necessarily come prepared with all the customer feedback like a Product RFC from our PM does. It might even skip this phase entirely, but still adapt the other phases.
But then if we reword this to make this more generic, the details and specificity on your RFC format are lost.

So honestly not sure which of your two options is more useful: To only document "Larger **product** features" and leave it implicit this can be a blueprint for other larger projects, or to document a generic process for all larger features with less specificity.

Reply:
I think that using 'driving individual' in all cases is better. It doesn't mean that the PM cannot participate in the RFC, it's only about driving the innitiative.

Reply:
@AlicjaSuska are you saying you would prefer to document a _generic_ process here and remove the expectations around summarizing user desires/feedback? (see my comment above)